src/prompt_generation/robustness_checks/choose_function.py

```diff
@@ -6,30 +6,24 @@
 Will use some work from pipelines.sequence_completions.py
 
 General format:
-SYSTEM_PROMPT + (BASE_PROMPT + BASE_PROMPT_COMPLETION (including answer)
-+ COT_PROMPT + COT_PROMPT_COMPLETION) * n_shot
```

Comment:
@ojorgensen please check whether the functions and prompts in this folder are used. If not, add a commit removing them or let me know.

Reply:
Removed the tokensiers file, but I think the rest of the prompt generation stuff is used.